Extensions/TextObject/textruntimeobject.ts

```diff
@@ -50,13 +57,18 @@ namespace gdjs {
     _textAlign: string = 'left';
     _wrapping: boolean = false;
     _wrappingWidth: float = 1;
-    _outlineThickness: number = 0;
-    _outlineColor: integer[] = [255, 255, 255];
-    _shadow: boolean = false;
-    _shadowColor: integer[] = [0, 0, 0];
-    _shadowDistance: number = 1;
-    _shadowBlur: integer = 1;
-    _shadowAngle: float = 0;
+
+    _isOutlineEnabled: boolean;
+    _outlineThickness: float;
+    _outlineColor: integer[];
+
+    _shadow: boolean;
+    _shadowColor: integer[];
+    _shadowOpacity: float;
+    _shadowDistance: float;
+    _shadowAngle: float;
+    _shadowBlur: float;
+
     _padding: integer = 5;
```

Comment:
No need to remove the default value here as well?

Reply:
The padding can't be set with properties.